<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tribal Check-ins</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_tribe"/>
    <addaction name="actionRemove_tribe"/>
    <addaction name="actionEdit_family_information"/>
    <addaction name="actionSet_active_inactive"/>
    <addaction name="actionAdd_Remove_checkin_methods"/>
    <addaction name="actionUndo_last_checkin"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOrder">
    <property name="title">
     <string>Order</string>
    </property>
    <addaction name="actionAlphabetical_order"/>
    <addaction name="actionLast_check_in_order"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_contents"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOrder"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_tribe">
   <property name="text">
    <string>Add tribe</string>
   </property>
   <property name="statusTip">
    <string>Add a new tribe to .db</string>
   </property>
  </action>
  <action name="actionAdd_Remove_checkin_methods">
   <property name="text">
    <string>Add/Remove check-in methods</string>
   </property>
   <property name="statusTip">
    <string>Add or remove a way tribes can checkin</string>
   </property>
  </action>
  <action name="actionExport_database">
   <property name="text">
    <string>Export database</string>
   </property>
  </action>
  <action name="actionAlphabetical_order">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Alphabetical order</string>
   </property>
   <property name="statusTip">
    <string>I want to see everything alphabetically</string>
   </property>
  </action>
  <action name="actionLast_check_in_order">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Last check in order</string>
   </property>
   <property name="statusTip">
    <string>I want to see everything chronologically</string>
   </property>
  </action>
  <action name="actionHelp_contents">
   <property name="text">
    <string>Help contents</string>
   </property>
   <property name="statusTip">
    <string>I don't know what's going on</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>What kind of thing is this thing?</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>I'm done with all this jazz, smell ya l8r</string>
   </property>
  </action>
  <action name="actionSet_active_inactive">
   <property name="text">
    <string>Set active/inactive</string>
   </property>
   <property name="statusTip">
    <string>Set which tribes require checkins and which are dormant</string>
   </property>
  </action>
  <action name="actionRemove_tribe">
   <property name="text">
    <string>Remove tribe</string>
   </property>
   <property name="statusTip">
    <string>Remove tribe and all logged checkins from .db</string>
   </property>
  </action>
  <action name="actionEdit_family_information">
   <property name="text">
    <string>Edit family information</string>
   </property>
   <property name="statusTip">
    <string>Change which families belong to which locations</string>
   </property>
  </action>
  <action name="actionUndo_last_checkin">
   <property name="text">
    <string>Delete last checkin</string>
   </property>
   <property name="statusTip">
    <string>&quot;Oops I shouldn't have clicked that&quot; - this is the option for you</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
